### PR TITLE
feat: added immutable tags section

### PR DESCRIPTION
**Pull request by tobiasweiss93**
Immutable tags documentation.

### CODE REVIEW

File: docs/meshcloud.metadata-tags.md [hunk 1]
```diff
@@ -27,6 +27,11 @@ Administrators can configure tags so that only [Partner users](./administration.
 
 The [administration documentation](./administration.index.md) provides further details on how partner users can edit restricted tags.
 
+## Immutable tags
+
+Administrators can configure immutable tags. This means that we allow the selection of specific tag values only during the creation process [meshCustomers](meshcloud.customer.md), [meshProjects](meshcloud.project.md), [meshLandingZones](meshcloud.landing-zones.md) and [meshPaymentMethods](./meshcloud.payment-methods.md). Afterwards it is no longer possible to edit the selected values. Also administrators cannot change immutable tags after creation.
```
Comment (uleMeshcloud) [inline docs/meshcloud.metadata-tags.md:32]: d: so our plan is once a tag is immutable we can unset  the immutable in tag catalogue right?
or what does it mean after creation?

File: docs/meshcloud.metadata-tags.md [hunk 2]
```diff
@@ -27,6 +27,11 @@ Administrators can configure tags so that only [Partner users](./administration.
```
Comment (uleMeshcloud) [inline docs/meshcloud.metadata-tags.md:27]: image
I would add immutable here as a description too

File: docs/meshcloud.metadata-tags.md (hunk 1 above)
Comment (uleMeshcloud) [reply docs/meshcloud.metadata-tags.md:32]: Just check the implementation, you are able to unset it.
```suggestion
Administrators can configure immutable tags. This means that we allow the selection of specific tag values only during the creation process [meshCustomers](meshcloud.customer.md), [meshProjects](meshcloud.project.md), [meshLandingZones](meshcloud.landing-zones.md) and [meshPaymentMethods](./meshcloud.payment-methods.md). Afterwards it is no longer possible to edit the selected values. Also administrators cannot change immutable tag values after creation.
```

**Review by uleMeshcloud — CHANGES_REQUESTED** [review]
I would alternate the other tag section too :)

File: docs/meshcloud.metadata-tags.md (hunk 1 above)
Comment (tobiasweiss93) [reply docs/meshcloud.metadata-tags.md:32]: You can remove the immutable flag from the tags. This works fine.

File: docs/meshcloud.metadata-tags.md (hunk 2 above)
Comment (tobiasweiss93) [reply docs/meshcloud.metadata-tags.md:27]: Fixed

**Review by sttomm — APPROVED** [review]

File: docs/meshstack.metadata-tags.md [hunk 3]
```diff
@@ -65,6 +65,7 @@ You will be prompted with a new page where you can enter a lot of information. W
 - **Default Value (optional)**: This can be used to give a default value to the user for contextual help or for filling in a default.
 - **Mandatory**: If a tag is mandatory, it means the user cannot create a new meshObject without first filling in this tag value.
 - **Restricted**: If a tag is restricted, it means only Partner users can enter this tag value. The input will also not be shown to the end-user when creating a new meshObject. This is useful if an organization doesn't want users to edit these values in self-service. For example, an organization can use restricted tags to implement a "segregation of duty" control to ensure that a partner user confirmed a project's data classification.
+- **Immutable**: If a tag is immutable, it means tag values can be assigned only during the creation of a meshObject. Aftwards it is not longer possible to change the tag values.
```
Comment (uleMeshcloud) [inline docs/meshstack.metadata-tags.md:68]:
```suggestion
- **Immutable**: If a tag is immutable, it means tag values can be assigned only during the creation of a meshObject. Afterwards it is not longer possible to change the tag values.
```

File: docs/meshcloud.metadata-tags.md [hunk 4]
```diff
@@ -27,6 +27,11 @@ Administrators can configure tags so that only [Partner users](./administration.
 
 The [administration documentation](./administration.index.md) provides further details on how partner users can edit restricted tags.
 
+## Immutable tags
+
+Administrators can configure immutable tags. This means that we allow the selection of specific tag values only during the creation process [meshCustomers](meshcloud.customer.md), [meshProjects](meshcloud.project.md), [meshLandingZones](meshcloud.landing-zones.md) and [meshPaymentMethods](./meshcloud.payment-methods.md). Afterwards it is no longer possible to edit the selected values. Also administrators cannot change immutable tag values after creation.
```
Comment (uleMeshcloud) [inline docs/meshcloud.metadata-tags.md:32]: Immutable tags can also be a string like coriaId  or I think from the term we can use set instead select to cover the different data types
```suggestion
Administrators can configure immutable tags. This means that we allow to set specific tag values only during the creation process [meshCustomers](meshcloud.customer.md), [meshProjects](meshcloud.project.md), [meshLandingZones](meshcloud.landing-zones.md) and [meshPaymentMethods](./meshcloud.payment-methods.md). Afterwards it is no longer possible to edit the values. Also administrators cannot change immutable tag values after creation.
```

**Review by uleMeshcloud — APPROVED** [review]
smaller remark but in general lgtm

File: docs/meshcloud.metadata-tags.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tobiasweiss93) [reply docs/meshcloud.metadata-tags.md]: Fixed.

File: docs/meshstack.metadata-tags.md (hunk 3 above)
Comment (tobiasweiss93) [reply docs/meshstack.metadata-tags.md:68]: Fxied.

File: docs/meshcloud.metadata-tags.md (hunk 4 above)
Comment (tobiasweiss93) [reply docs/meshcloud.metadata-tags.md:32]: Please take a look into the PR https://github.com/meshcloud/meshfed-release/pull/2377

**Review by chewbeccy — APPROVED** [review]

**Pull request merged**